vfs/fs_utils.h

```diff
@@ -48,6 +49,8 @@ struct FsObjectInfo {
     std::map<std::string, Json::Value> objectMetadata;  ///< Metadata from obj
 
     operator bool () const { return exists; }
+
+    std::exception_ptr excPtr;  ///< MLDB-2150, hack, do not use
```

Comment:
If we make close() work properly, we won't need this and/or we can make an "error" field that's just a string.